frame/system/src/lib.rs

```diff
@@ -847,6 +847,27 @@ impl<O, T> EnsureOrigin<O> for EnsureNever<T> {
 	}
 }
 
+pub struct EnsureOneOf<AccountId, A, B>(A, B, sp_std::marker::PhantomData<AccountId>);
```

Comment:
Docs are missing. 

You don't need `AccountId` as generic parameter here:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=52be76900f8b1dc497681e1c1e33f352

Reply:
In this way `L` and `R`'s `EnsureOrigin` bound would be `EnsureOrigin<AccountId>`, instead of `EnsureOrigin<O>` with `O: Into<Result<RawOrigin<AccountId>, O>> + From<RawOrigin<AccountId>>` like other `EnsureX` impl. But it should be fine. Will update.

Reply:
Okay, you are right. I did not realize this. So this is okay! Sorry!